projectroles/views.py

```diff
@@ -2901,6 +2908,27 @@ def form_valid(self, form):
         )
         return redirect(reverse('projectroles:remote_sites'))
 
+    def create_timeline_event(
+        self, remote_site, user, form_action, timeline=None
+    ):
+        """Create timeline event for remote site creation/update"""
+        status = form_action if form_action == 'set' else form_action[0:-1]
+        if remote_site.mode == SITE_MODE_SOURCE:
+            event_name = 'source_site_{}'.format(status)
+        else:
+            event_name = 'target_site_{}'.format(status)
+        if timeline:
+            timeline.add_event(
+                project=None,
```

Comment:
Setting the event status is missing, hence these events appear with the `INIT` status in timeline. You should set the `OK` status when creating the event.